specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,615 @@
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string"
+        },
+        "description": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string",
+          "maxLength": 256
+        },
+        "maxScore": {
+          "description": "Maximum control score (0..10)",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 10,
+          "exclusiveMinimum": false
+        },
+        "source": {
+          "description": "Source object from which the control was created",
+          "readOnly": true,
+          "$ref": "#/definitions/SecureScoreControlDefinitionSource"
+        },
+        "assessmentsDefinition": {
+          "description": "array of assessments metadata ids",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDefinitionAssessmentsId"
+          }
+        }
+      }
+    },
+    "SecureScoreControlDefinitionAssessmentsId": {
+      "type": "object",
+      "description": "Secure Score Control Definition Properties",
+      "properties": {
+        "id": {
+          "description": "Full assessment metadata azure ids that build the control",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoreControlDefinitionSource": {
+      "description": "representing the source of the control",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "sourceType": {
+          "description": "BuiltIn if the control is built-in from Azure Security Center managed assessments, Custom if the assessment based on custom Azure Policy definition, CustomerManaged for customers who build their own controls",
+          "type": "string",
+          "enum": [
+            "BuiltIn",
+            "CustomPolicy",
+            "CustomerManaged"
+          ],
+          "x-ms-enum": {
+            "name": "controlType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "BuiltIn",
+                "description": "Azure Security Center managed assessments"
+              },
+              {
+                "value": "CustomPolicy",
+                "description": "User defined policies that are automatically ingested from Azure Policy to Azure Security Center"
+              },
+              {
+                "value": "CustomerManaged",
+                "description": "User controls pushed directly by the user or other third party to Azure Security Center"
+              }
+            ]
+          }
+        },
+        "policySetDefinitionId": {
+          "description": "id of the policy set definition id from which the control was generated",
+          "type": "string"
+        },
+        "groupName": {
```

Comment:
will it be relevant to all scenarios?

Reply:
Removing it. To be added once Custom controls are added